### PR TITLE
doc: Update console branding instructions for cloud

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Closes https://github.com/TheThingsIndustries/lorawan-stack-support/issues/268

#### Screenshots

Screenshot 2021-01-11 at 16 09 19

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/reference/branding/_index.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ This reference gives details on how to customize the branding of the login pages
 
 <!--more-->
 
+## Cloud Branding
+
+Cloud users need to [contact The Things Industries](mailto:support@thethingsindustries.com) to update branding in the console. See below for what's available to customize and what files are necessary. The files can be uploaded to a CDN or sent to The Things Industries to be uploaded.
```
Comment (kschiffer) [inline doc/content/reference/branding/_index.md:12]:
```suggestion
Cloud users need to [contact The Things Industries](mailto:support@thethingsindustries.com) to update branding in the console. See below for what's available to customize and what files are necessary. To request a branding update, either send us a link to a CDN which hosts these files or attach them to the email.
```

**Review by kschiffer — APPROVED** [review]
👍 
Note @htdvisser's update.

**Comment by benolayinka** [comment]
> 👍
> Note @htdvisser's [update](https://github.com/TheThingsIndustries/lorawan-stack/pull/2508).

okay but there is still no CLI function for updating console options at https://www.thethingsindustries.com/docs/reference/configuration/console/

correct?

**Comment by kschiffer** [comment]
Yeah, I guess we can merge this now and amend it later.

**Pull request merged**